cmd/immudb/command/init.go

```diff
@@ -37,6 +37,8 @@ func (cl *Commandline) setupFlags(cmd *cobra.Command, options *server.Options) {
 	cmd.PersistentFlags().StringVar(&cl.config.CfgFn, "config", "", "config file (default path are configs or $HOME. Default filename is immudb.toml)")
 	cmd.Flags().String("pidfile", options.Pidfile, "pid path with filename. E.g. /var/run/immudb.pid")
 	cmd.Flags().String("logfile", options.Logfile, "log path with filename. E.g. /tmp/immudb/immudb.log")
+	cmd.Flags().String("logformat", options.LogFormat, "log format. E.g. text/json")
```

Comment:
what about "log format e.g. text/json" ?
If so, we may update the message in other flags

Reply:
ack